tmp/nicholasng-me-lint/index.html

```
@@ -0,0 +1,503 @@
+
+<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)